README.md

```diff
@@ -37,11 +37,11 @@ If you do not have nvm (node version manager), first install that: https://githu
 
     $ nvm install && nvm use # To get on the right node version
     $ make setup
-    $ make build
 
 Follow the instructions under [Finance disclosures](#finance-disclosures).
 
     $ make pull-finance
+    $ make build
```

Comment:
Thanks! As it stands, `make build` is actually optional since `make serve` runs the build and then spins up the site. We could have it say `make build (optional)` or something, or make another note somewhere mentioning this. I can merge this for the sake of accuracy for now, but I'm also planning to update the readme overall in the next week or so.